### DISCUSSION TITLE
Text going off the page in ePub

### DISCUSSION THREAD

**Discussion by kaelemc** [sumatrapdfreader/sumatrapdf#3646] [Q&A]
Hi, 

In the attached image you can see there is some text that is going off the page, I've tried a few other readers that can handle ePub and they place a scroll bar within that snippet so that I can scroll.

Is there a way I can change page width or is there any roadmap to add customisation for epub?, As far as I understand eBookUI was removed in 3.4.

Thanks.

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/62122480/26d7060d-88f0-42a1-a607-98f29a34a5d0)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/62122480/572a16b2-f406-41c9-b24c-2b998b0cad87)

**Comment by kjk** [comment]
Unfortunately it's unlikely to be fixed given that is how mupdf epub rendering works.
